var/spack/repos/builtin/packages/chill/package.py

```diff
@@ -0,0 +1,101 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#-----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+#-----------------------------------------------------------------------------
+from spack import *
+
+
+class Chill(Package):
+    """FIXME: Put a proper description of your package here."""
+    rose_git        = "https://github.com/rose-compiler/rose-develop.git"
+    rose_version    = "v0.9.10.0"
+    
+    homepage    = "http://github.com/CtopCsUtahEdu"
+    url         = "https://github.com/CtopCsUtahEdu/chill/archive/v0.3.tar.gz"
+    git         = "https://github.com/CtopCsUtahEdu/chill.git"
+
+    #version('0.3', sha256='574b622368a6bfaadbe9c1fa02fabefdc6c006069246f67d299f943b7e1d8aa3')
+    version('master', branch='master')
+
+    depends_on("autoconf@2.69:", type='build')
+    depends_on("automake@1.14:", type='build')
+    depends_on("libtool@2.4:", type='build')
+    depends_on("bison", type='build')
+    depends_on("flex", type='build')
+    depends_on("boost@1.60.0:1.69.0")
+    depends_on('iegenlib')
+    depends_on('isl')
+    depends_on('python')
+    
+    def autoreconf(self, spec, prefix):
+        bash = which('bash')
+        
+        with working_dir(self.stage.source_path + '/..'):
+            bash('git clone -b {0} {1}'.format(rose_version, rose_git, 'rose_src')
```

Comment:
This needs to be converted to a dependency or a resource.

Reply:
I might send a new merge request with a 'rose-for-chill' package.
I'll update the license as well.

Reply:
new pull request: https://github.com/spack/spack/pull/11406